src/main/java/net/dv8tion/jda/internal/requests/restaction/interactions/MessageEditCallbackActionImpl.java

```diff
@@ -68,7 +65,7 @@ protected DataObject toData()
             data.put("components", DataArray.fromCollection(components));
         if (retainedFiles != null)
         {
-            json.put("attachments", DataArray.fromCollection(
+            data.put("attachments", DataArray.fromCollection(
```

Comment:
Nice. Was gonna follow up with you on this one.

Reply:
Btw, Lets put a comment on this indicating that it isn't actually being used for now. I know we commented out the public method that would run this code-path, but I think we should mention that this isn't in use even still.